README.md

```diff
@@ -138,6 +138,14 @@ render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
 
 `meta` will only be included in your response if you are using an Adapter that supports `root`, as JsonAPI and Json adapters, the default adapter (FlattenJson) doesn't have `root`.
 
+### Links
```

Comment:
@bf4 agree with you. I could improve the code to reflect this design and also refactor the method `attributes_for_top_level_links` 

@joaomdmoura better to do it after merge #1041, to avoid conflicts ?